tests/main/confinement-classic/task.yaml

```diff
@@ -4,7 +4,10 @@ details: |
     classic confinement can be executed correctly. There are two variants of
     this test (classic and jailmode) and the snap (this particular one) should
     function correctly in both cases.
-systems: [-ubuntu-core-16-*,-debian-*]
+systems:
+    - -ubuntu-core-16
+    # No confinement (AppArmor, Seccomp) available on these systems
```

Comment:
Exactly that is the plan but in a second wave. As soon as all tests are green I want a first version landed. Using a `snap is-forced-devmode` check will then require some refactoring in the tests which I want clearly separate from this to not make the PRs bigger as needed.

This is classic confinement but fails because of jailmode. Lets enable this again in the second wave. Its not like I am throwing a PR and will never care again :-)